V/ViennaRNA/build_tarballs.jl

```diff
@@ -104,6 +113,44 @@ fi
     -Wl,$(flagon --whole-archive) ./src/ViennaRNA/libRNA.a -Wl,$(flagon --no-whole-archive) \
     $libs
 
+
+# compile RNAxplorer, we do this separately because it links to
+# BLAS/LAPACK, and PCRE on windows
+
+LIBS_RNAxplorer="$LIBS"
+if [[ "${target}" == *-w64-mingw32* ]]; then
+    # work around there not being a regex.h on w64-mingw32
+    cp "${includedir}/pcreposix.h" "${includedir}/regex.h"
+    LIBS_RNAxplorer="$LIBS_RNAxplorer -lpcreposix-0"
+
+    # BLAS/LAPACK support
+    LIBS_RNAxplorer="$LIBS_RNAxplorer -lblastrampoline-5"
```

Comment:
Uhm, I'm a bit confused, I don't see libblastrampoline.  Also, `-lblastrampoline-5` on Windows requires lbt v5.4.0 (I think) which is only compatible with Julia v1.9+.

Reply:
> Uhm, I'm a bit confused, I don't see libblastrampoline.

I need `LAPACK_jll` to build the RNAxplorer executable, and `liblapack.so` from `LAPACK_jll v3.11.0+0` links to `libblastrampoline.so.5`.  That's why I need to add `-lblastrampoline` to `$LIBS_RNAxplorer`.

> Also, `-lblastrampoline-5` on Windows requires lbt v5.4.0 (I think) which is only compatible with Julia v1.9+.

So use `julia_compat="1.6"` in the call to `build_tarballs`?